workflow/overview.mdx

```diff
@@ -0,0 +1,176 @@
+
+When a trigger occurs, it sends the necessary payload data that the workflow will use to deliver notifications to subscribers. **Learn more about subscribers**.
+
+Triggers are the primary and only method for sending notifications. The trigger identifier ensures that the correct workflow is executed.
+
+**Example of a Trigger**
+Here's an example of an API call to trigger an event:
+
+```bash
+curl -X POST https://api.novu.co/v2/events/trigger \
+  -H "Authorization: ApiKey <NOVU_SECRET_KEY>" \
+  -H "Content-Type: application/json" \
+  -d '{
+    "name": "<WORKFLOW_TRIGGER_IDENTIFIER>",
+    "to": {
+      "subscriberId": "<UNIQUE_SUBSCRIBER_IDENTIFIER>",
+    },
+    "payload": {}
+    }
+  }'
+```
+
+### Channel Step
+
+A **Channel Step** in Novu represents a configured provider to send notifications to your recipients.
+
+The Workflow Editor is designed for **Omni-Channel Communication**, delivering a unified experience across diverse messaging channels:
+
+- **In-App Notifications**
+- **Email**
+- **SMS**
+- **Push**
+  - **Mobile Push**
+  - **Web Push**
+- **Chat**
+  - **Enterprise Messaging Platforms** (e.g., Slack, Microsoft Teams)
+  - **Consumer Messaging Apps** (e.g., WhatsApp, Telegram, Discord)
+
+Most providers within Novu use credentials that you supply to deliver notifications on your behalf. These credentials and other settings are what make a configured channel. 
+
+### Action Step
+
+**Action Steps** are purpose built functions that help manage the workflow flow. **Action Steps** can be executed before and after **Channel Steps**. 
+For example, you can use a **Delay** to pause a workflow for a certain duration before sending a notification.
+
+There are three types of Action Steps supported by Novu:
+
+- Delay
+- Digest
+- Custom
+
+## Key Features
+
+- **Failure Resilience**
+  - Ensures workflows resume from point of failure during platform outages
+  - Guarantees processing of critical notifications in unstable environments
+
+- **Handling Long-Running Tasks**
+  - Manages time-sensitive processes like batched notifications and digest emails
+  - Optimized for platforms with strict execution time limits
+  - Ideal for serverless environments
+
+- **Event-Driven Workflow Management**
+  - Supports workflow pausing until external events occur
+  - Perfect for scenarios like:
+    - Waiting for user actions (e.g., confirmation link clicks)
+    - Responding to asynchronous notifications from integrated services
+
+- **Scheduled Notifications**
+  - Automates recurring communication
+  - Supports cron-based scheduling
+  - Easily manage reminders, routine updates
+
+- **Parallel Notification Delivery**
+  - Sends multiple notifications simultaneously
+  - Ensures successful completion of all notifications
+  - Minimizes delays in multi-channel communication
```

Comment:
I'm not sure about this whole section, a lot of very generic bullets and words here. I feel like it should be part of the "Building workflows" section, especially when you already have a usecases section at the bottom